### PR TITLE
Update Docs 

### PR DESCRIPTION
**Pull request by cheeerynne**
- Update AboutUs
- Update DG (Use cases, Storage diagram, clearAssignee implementation)
- Update UG (Minor typos)

### CODE REVIEW

File: docs/team/cheeerynne.md [hunk 1]
```diff
@@ -11,62 +11,64 @@ that are typed into the input box as commands and it has a GUI created with Java
...
     * Justification: This feature improves the product significantly because a user can easily filter and find tasks based on specific 
       keywords. With this easy filtering, there is no need for users to scroll through the entire list just to find a specific task. 
-    * Highlights: The implementation was challenging as I had to check only the title and description fields of a task. 
-      I decided to implement a predicate, which takes in the title and description of a task, to check if any of these 2 fields
-      matches any of the keywords given. Furthermore, integration and unit testing had to be implemented thoroughly to ensure 
-      that the various scenarios of usage were accounted for.
-
-**New Feature**: Added the ability to find members with specified keywords. (Pull Requests: [\#130](https://github.com/AY2021S2-CS2103T-W14-3/tp/pull/130))
-    * What it does: allows the user to find all members whose name, phone number, email or role any of the specified keywords.
+    * Highlights: The implementation was challenging as I had to check through the title and description field of all the tasks
```
Comment (zatkiller) [inline docs/team/cheeerynne.md]: i think it should be fields

File: docs/team/cheeerynne.md [hunk 2]
```diff
@@ -11,62 +11,64 @@ that are typed into the input box as commands and it has a GUI created with Java
...
       keywords. With this easy filtering, there is no need for users to scroll through the entire list just to find a specific task. 
-    * Highlights: The implementation was challenging as I had to check only the title and description fields of a task. 
-      I decided to implement a predicate, which takes in the title and description of a task, to check if any of these 2 fields
-      matches any of the keywords given. Furthermore, integration and unit testing had to be implemented thoroughly to ensure 
-      that the various scenarios of usage were accounted for.
-
-**New Feature**: Added the ability to find members with specified keywords. (Pull Requests: [\#130](https://github.com/AY2021S2-CS2103T-W14-3/tp/pull/130))
-    * What it does: allows the user to find all members whose name, phone number, email or role any of the specified keywords.
+    * Highlights: The implementation was challenging as I had to check through the title and description field of all the tasks 
+      in the task list. I decided to implement a predicate,TaskContainsKeywordPredicate, which compares the title and description of
```
Comment (zatkiller) [inline docs/team/cheeerynne.md]: space before TaskContainsKeywordPredicate

File: docs/team/cheeerynne.md [hunk 3]
```diff
@@ -11,62 +11,64 @@ that are typed into the input box as commands and it has a GUI created with Java
...
+      a task to the list of keywords specified. The task list is then updated accordingly to the predicate.
```
Comment (zatkiller) [inline docs/team/cheeerynne.md:30]: Should refer to the filtered task list for more clarity

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -377,15 +376,15 @@ Use case ends.
 
 **MSS**
```
Comment (zatkiller) [inline docs/DeveloperGuide.md:378]: Might be better to indicate that the list is the filtered / currently displayed list of members. (As opposed to full list of members)

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -394,20 +393,20 @@ Use case ends.
...
```
Comment (zatkiller) [inline docs/DeveloperGuide.md:394]: Similar to deleteMember. Might be better to indicate that the list is the filtered / currently displayed list of members. (As opposed to full list of members)

**Review by zatkiller — CHANGES_REQUESTED** [review]
Might want to make some points clearer to prevent ambuigity. Other than that can merge after changes made

File: docs/UserGuide.md [hunk 6]
```diff
@@ -176,7 +176,7 @@ As members leave the club, you can delete a member along with all of his/her con
 Format: `deleteMember NAME`
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-The NAME you specify must be in the currently displayed list of Members  
+The NAME specified is case-sensitive and must be in the current displayed list of Members.
```
Comment (zatkiller) [inline docs/UserGuide.md]: I think we can make this clearer by making it "The NAME specified must be in the currently displayed list of Members when executing this command". We also specified above the NAME field is case-sensitive so I guess we don't have to repeat it?

File: docs/UserGuide.md [hunk 7]
```diff
@@ -200,14 +200,12 @@ existing Member in the application with the name you specify.
...
 * At least one of the optional fields must be provided.
-* NEW_ROLE field can take on any values with alphanumeric characters.
-* NEW_NAME field can take on any values with alphanumeric characters.    
-* Existing values will be updated to the input values.
+* NEW_NAME and NEW_ROLE fields can take on any values with alphanumeric characters.
+* Existing values will be updated to the input values. 
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-The NAME you specify must be in the currently displayed list of Members 
+The NAME specified is case-sensitive and must be in the currently displayed list of Members
```
Comment (zatkiller) [inline docs/UserGuide.md]: Should keep the formatting consistent. Put case sensitive above too or keep it as such

File: docs/team/cheeerynne.md (hunk 3 above)
Comment (cheeerynne) [reply docs/team/cheeerynne.md:30]: findTasks does not check through the filtered task list, it checks through the unfiltered task list.

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (cheeerynne) [reply docs/DeveloperGuide.md:394]: Okay

**Review by zatkiller — APPROVED** [review]
Can merge after making changes

**Pull request merged**